client/src/pages/Plan/Plan.js

```diff
@@ -16,6 +16,28 @@ const Plan = () => {
     const { student, realCourses, programme } = usePlan();
     const [searchTerm, setSearchTerm] = useState("");
     const [note, setNote] = useState("");
+    const start = 2020;
+    const end = 2021;
+    const years = [];
+    for (var i = start; i <= end; i++) {
+        years.push(i);
+    }
+    console.log(years);
+    const [courseAllocations, setCourseAllocations] = useState([
+        { course: "Softeng 251", year: 2020, semester: "S1" },
+        { course: "Softeng 252", year: 2020, semester: "S1" },
+        { course: "Softeng 253", year: 2020, semester: "S1" },
+        { course: "Softeng 254", year: 2020, semester: "S1" },
+        { course: "Softeng 255", year: 2020, semester: "S2" },
+        { course: "Softeng 256", year: 2020, semester: "S2" },
+        { course: "Softeng 351", year: 2021, semester: "S1" },
+        { course: "Softeng 352", year: 2021, semester: "S1" },
+        { course: "Softeng 353", year: 2021, semester: "S1" },
+        { course: "Softeng 354", year: 2021, semester: "S1" },
+        { course: "Softeng 451", year: 2021, semester: "S2" },
+        { course: "Softeng 452", year: 2021, semester: "S2" },
+        { course: "Softeng 453", year: 2021, semester: "S2" },
+    ]);
```

Comment:
is there  a backend for this or are u gonna do that on the other ticket

Reply:
Yea already part of a plan object